src/actions/sessionActions.js

```diff
@@ -7,7 +7,7 @@ export const login = user =>
   async () => {
     try {
       const response = await sessionApi.login({ user });
-      sessionService.saveUser(response.user);
+      sessionService.saveUser(response.data);
```

Comment:
And then use the data object
```suggestion
      sessionService.saveUser({ data });
```

Reply:
Good call!